src/uct/ib/rc/base/rc_iface.c

```diff
@@ -78,6 +82,16 @@ ucs_config_field_t uct_rc_iface_config_table[] = {
    ucs_offsetof(uct_rc_iface_config_t, tm.list_size), UCS_CONFIG_TYPE_UINT},
 #endif
 
+#if HAVE_IBV_EXP_DM
+  {"DM_SEG_SIZE", "0",
+   "Device Memory segment size (0 - disabled)",
```

Comment:
what if not 0?  What range is allowed?

Reply:
depends from available Device Memory

Reply:
Seems like this is user's responsibility to organize sharing... how he can know total available memory?
Can we somehow use ucp_params_t::estimated_num_eps hint or add new one?

Reply:
> we can add code to evaluate total amount of DM, but we can't pass this information to user (how???).

ucx_info?

> current implementation is enough flexible to configure DM manually for wide scope of tasts

yes for you and useless for user if he doesn't know details.

Reply:
will discuss